.github/workflows/test.yaml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     container:
-      image:  python:3.6-stretch
+      image:  python:3.9-buster
```

Comment:
stretch is not available as an image for 3.9 and alpine failed on CI test FYI, I'm guessing this is why stretch was used instead of alpine for the original 3.6 image

Reply:
Would you be able to update this job to run tests without a container?

I did a similar thing in this PR https://github.com/cds-snc/notification-api/pull/1192 and bumped our Postgres version to match staging/production

Reply:
Sure, put that code in there and I left the matrix with a single item list in to simplify future upgrade tests etc.